snuba/cli/consumer.py

```diff
@@ -46,6 +46,7 @@
     "storage_name",
     type=click.Choice(
         [storage_key.value for storage_key in get_writable_storage_keys()]
+        + ["search_issues"]
```

Comment:
Temporary workaround to enable running the consumer. Will remove this when the choices for storage options load from configs properly.

Reply:
cc @evanh

Reply:
Reading though https://github.com/getsentry/snuba/blob/master/snuba/datasets/storages/factory.py#L132, it seems like this should not be needed as long as you turn on ENABLE_DEV_FEATURES and add it to the dev storages list

Reply:
This PR should fix this bug: https://github.com/getsentry/snuba/pull/3502

Reply:
Looking at the factory and how its initialized, I can't tell how to register a storage as a `_dev_writable_storages`. 

Could we keep this workaround for this PR since it looks like that PR is still awaiting a merge to master.

We're looking to get this PR merged to start testing the whole flow within sentry.